### PR TITLE
Fix scrollbar's position and pointers-event

### LINKED ISSUE #16: Can't move scrollbar on Chrome (Linux?)
On Latest Chrome you can't use the scrollbar to move the page down, because the scrollbar is sitting behind the header and trying to click it just doesn't work.

At least on Linux.

### PR DESCRIPTION
**Pull request by oldergod**
fix #16

### CODE REVIEW

File: src/styles/core.scss [hunk 1]
```diff
@@ -42,7 +42,7 @@ body {
 .app-container {
   overflow-x: hidden;
   overflow-y: scroll;
-  padding-top: 56px;
+  margin-top: 56px;
```
Comment (oldergod) [inline src/styles/core.scss]: so scrollbar appears after the header

File: src/styles/core.scss [hunk 2]
```diff
@@ -53,10 +53,12 @@ body {
   &.locked {
     overflow-x: hidden;
     overflow-y: hidden;
+    margin-top: 0;
```
Comment (oldergod) [inline src/styles/core.scss]: to avoid display bugs of the close button on not `min-width: 645px`

File: src/styles/core.scss [hunk 3]
```diff
@@ -53,10 +53,12 @@ body {
...
   }
 }
 
 .app-header {
+  pointer-events: none;
```
Comment (oldergod) [inline src/styles/core.scss]: so can click scrollbar from its top

File: src/styles/core.scss [hunk 4]
```diff
@@ -86,11 +88,15 @@ body {
     z-index: 0;
   }
 
-  &.extended::before {
-    transform: translateY(0);
+  &.extended {
+    pointer-events: auto;
```
Comment (oldergod) [inline src/styles/core.scss]: so clicked on the extended header do not open the property behind

**Comment by surma** [comment]
Thanks @oldergod for the awesome work. 

There’s only one issue (that I ran into as well while trying to whip up a quick fix) when the browser window gets small:

screenshot 2016-05-09 12 49 33

Do you have an idea on how to fix this?

**Comment by oldergod** [comment]
@surma I cannot repro this. I tried on Chrome/Firefox(Ubuntu), Safari/Chrome(iPhone6S), Chrome(Nexus5)
Here is what I see in Chrome/Firefox on Ubuntu:

image

Am I missing something?
Since I see a ruler, I guess you are using Chrome? on Mac? I'll be able to try this on Mac at the end of the week.

**Comment by oldergod** [comment]
I could reproduce what you saw this way:

  1. toggle device mode
  2. set device type to Desktop
  3. refresh and scroll a bit till header expands
  4. set device type to Mobile

But the werid thing is : if I start with a device type set to Mobile, I cannot reproduce it. I tested on iPhone6s and Nexus5 without problem so I would rather think it is a chrome thing?

css_triggers_test_scrollbar

**Comment by surma** [comment]
Alright, if it’s a chrome-thing only, then feel free to ignore it.

Is this ready to merge in your opinion?

**Comment by oldergod** [comment]
IMO it is, yes. If ever a bug were to show up, I would promptly fix it anyway!

**Comment by surma** [comment]
@oldergod Forgive me for how long it took me to get back to you! I/O and the subsequent jetlag knocked me out ;)

Finally gave this the last test run. LGTM. Merged. Awesome 🎉

**Pull request merged**